tests/by-util/test_chmod.rs

```diff
@@ -233,13 +233,20 @@ fn test_chmod_ugoa() {
 
 #[test]
 #[cfg(any(target_os = "linux", target_os = "macos"))]
```

Comment:
I guess you made the changes to make the test work on Android. And so I think you can either remove the `cfg` or add a `target_os = "android"`.